loom/common/core/log.c

```diff
@@ -47,6 +47,15 @@ typedef struct loom_log_listenerEntry
     struct loom_log_listenerEntry *next;
 } loom_log_listenerEntry_t;
 
+typedef struct loom_log_bufferEntry
+{
+    loom_logGroup_t *group;
+    loom_logLevel_t level;
+    // Don't forget to free!
+    const char *msg;
```

Comment:
That's a good idea, however, currently I'm just reusing `lmLogArgs` which handles the all the varargs logic and buffer allocation and just use the buffer it returns and I'd rather not complicate that too much. Since the buffer gets only used on startup, I don't think it's a big problem?